src/components/HarborWidget/HarborWidget.tsx

```diff
@@ -15,7 +15,7 @@ const Widget = ({ entity }: { entity: Entity }) => {
 
   return (
     <Card>
-      <CardHeader title="Vulnerabilities in Docker Images" />
```

Comment:
I would rather change it to Container Images, as docker is a more product specific.

Reply:
anyway I ma happy to merge in.